music.sql

```diff
@@ -0,0 +1,90 @@
+CREATE DATABASE playlist;
```

Comment:
While this may have been helpful during setup, the migrations are expected to do the database setup work and this file should not be included in your repo.